timelock-impl/src/main/java/com/palantir/atlasdb/timelock/auth/api/Authorizer.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.timelock.auth.api;
+
+import com.palantir.atlasdb.keyvalue.api.Namespace;
```

Comment:
Hmm, I think we're not using the right construct here. This `Namespace` object is at a layer below the atlas keyspace/db-name/sid, which products can use to create multiple table-references with the same table name (which can be useful if you're using libraries that want to create tables and you want to avoid clashes), so it doesn't seem right.

There can exist namespace objects that aren't valid timelock namespaces, for example `@!@#(*$#@%&*` is a valid `UNCHECKED_NAME` `Namespace` object.